### ISSUE TITLE
`nodesCreated` "Int cannot represent non-integer value"

### ISSUE THREAD

**Issue by darrellwarde** [neo4j/graphql#1104]
**Describe the bug**

**Type definitions**

```gql
type Parent {
  id: ID! @id
  name: String
  child: Child! @relation(name: "HAS_CHILD" direction: OUT)
}

type Child {
  id: ID! @id
  name: String
}
```

**To Reproduce**

Run Mutation:

```gql
mutation CreateParentAndChild {
createParents(
input: {name: "parent-1", child: {create: {node: {name: "child-1"}}}}
) {
parents {
id
child {
id
}
}
info {
nodesCreated
relationshipsCreated
}
}
}
```

See the error:

```json
{
"errors": [
{
"message": "Int cannot represent non-integer value: { low: 2, high: 0 }",
"locations": [
{
"line": 19,
"column": 7
}
],
"path": [
"createParents",
"info",
"nodesCreated"
],
"extensions": {
"code": "INTERNAL_SERVER_ERROR",
"exception": {
"message": "Int cannot represent non-integer value: { low: 2, high: 0 }"
}
}
}
],
"data": null
}
```

**Expected behavior**

No error.

**System (please complete the following information):**

* Version: `@neo4j/graphql@2.5.6`

**Comment by neo4j-team-graphql** [comment]
Many thanks for raising this bug report @darrellwarde. :bug: We will now attempt to reproduce the bug based on the steps you have provided.

Please ensure that you've provided the necessary information for a minimal reproduction, including but not limited to:
* Type definitions
* Resolvers
* Query and/or Mutation (or multiple) needed to reproduce

If you have a support agreement with Neo4j, please link this GitHub issue to a new or existing Zendesk ticket.

Thanks again! :pray:

**Comment by tbwiss** [comment]
This error can be resolved by upgrading to version 3.0.X
Closing this bug report.
